protocol/x/clob/keeper/clob_pair.go

```diff
@@ -589,6 +637,19 @@ func (k Keeper) validateInternalOperationAgainstClobPairStatus(
 			clobPairId,
 			types.ClobPair_STATUS_INITIALIZING,
 		)
+	case types.ClobPair_STATUS_FINAL_SETTLEMENT:
+		// Only allow deleveraging events. This allows the protocol to close out open
+		// positions in the market. All trading is blocked.
```

Comment:
Suggestion: is this more accurate?

```suggestion
		// Only allow deleveraging events. This allows the protocol to close out open
		// positions in the market. All other operations are not allowed.
```